aiidalab_launch/__main__.py

```diff
@@ -229,14 +229,19 @@ def add_profile(ctx, app_state, port, home_mount, image, profile):
 @click.argument("profile")
 @click.option("--yes", is_flag=True, help="Do not ask for confirmation.")
 @click.option("-f", "--force", is_flag=True, help="Proceed, ignoring any warnings.")
+@click.option("--purge", is_flag=True, help="Remove all data associated with profile.")
 @pass_app_state
-def remove_profile(app_state, profile, yes, force):
+@click.pass_context
+def remove_profile(ctx, app_state, profile, yes, force, purge):
     """Remove an AiiDAlab profile from the configuration."""
     try:
         profile = app_state.config.get_profile(profile)
     except ValueError:
         raise click.ClickException(f"Profile with name '{profile}' does not exist.")
     else:
+        if purge:
```

Comment:
Hmm, wait a minute. I believe this code should be called at the very end right? Only after the profile has been successfully deleted we should delete the volumes. Also please add a specific WARNING and confirmation for this case.